react/NewsletterContext.tsx

```diff
@@ -144,6 +157,13 @@ function newsletterContextReducer(state: State, action: Action): State {
       }
     }
 
+    case 'SET_CUSTOM_VALUES': {
```

Comment:
VTEX IO apps only export the `export default` of entrypoints. I think we have to change this file